frontend/src/api/index.ts

```diff
@@ -7,18 +7,20 @@ const openInNewTab = (url: string) => {
 };
 
 export const login = async (): Promise<void> => {
-  return fetch(`http://localhost:5000/auth/login`).then(async (response) => {
-    const redirectUrl = await response.text();
-    openInNewTab(redirectUrl);
-  });
+  return fetch(`http://${process.env.HOST}:5000/auth/login`).then(
+    async (response) => {
+      const redirectUrl = await response.text();
+      openInNewTab(redirectUrl);
+    }
+  );
 };
 
 export const getPlaylists = async (
   offset: number,
   limit: number
 ): Promise<Playlist[]> => {
   const response = await fetch(
-    `http://localhost:5000/?limit=${encodeURIComponent(
+    `http://${process.env.HOST}:5000/?limit=${encodeURIComponent(
```

Comment:
Minor, but rather than having the port hardcoded, should we actually just bake that into the host? E.g. if we release to a real site, then presumably we'll eventually host on port 443 (https default)

Reply:
I've kept them separate as host and port as I provide them separately to esbuild. But good shout, have made the port an env variable.